_i18n/pt/guide.md

```diff
@@ -0,0 +1 @@
+Please translate me with awesome markdown
```

Comment:
@carmem-s This is the main file to edit, and you can look at `_i18n/en/guide.md` as inspiration